static/js/containers/UserPage_test.js

```diff
@@ -149,7 +149,7 @@ describe("UserPage", () => {
 
       it('should let you add an education entry', done => {
```

Comment:
Or if you'd rather merge this first that would be fine, we can do it as a separate issue